src/feedback/trulens/feedback/v2/feedback.py

```diff
@@ -187,6 +232,51 @@ class Coherence(Semantics):
     )
 
 
+@dataclass
+class CriteriaOutputSpaceMixin:
+    system_prompt: ClassVar[str]
+    output_space_prompt: ClassVar[str]
+    system_prompt_template: ClassVar[str]
+    criteria_template: ClassVar[str]
+
+    @staticmethod
+    def validate_criteria_and_output_space(criteria: str, output_space: str):
+        validated = EvalSchema(criteria=criteria, output_space=output_space)
+        return validated
+
+    @classmethod
+    def override_criteria_and_output_space(
```

Comment:
shall we rename this to sth along the line of `generate_system_prompt_with_criteria_and_output_space` (or sth less verbose but more explicit?)